versioned_docs/version-1.21.1/resources/server/datamaps/builtin.md

```diff
@@ -224,5 +224,30 @@ Example:
 }
 ```
 
+
+## `neoforge:villager_types`
+
+Allows configuring the villager type that will spawn when in a certain biome, as a replacement for `VillagerType#BY_BIOME`. It is located at `neoforge/data_maps/worldgen/biome/villager_types.json` and its objects have the following structure:
```

Comment:
```suggestion
Allows configuring the villager type that will spawn based on its biome, as a replacement for `VillagerType#BY_BIOME` (which will be ignored in 1.22). It is located at `neoforge/data_maps/worldgen/biome/villager_types.json` and its objects have the following structure:
```

Added a message saying that the fallback is removed in 1.22 and changed the wording of the spawn condition slightly.